lib/datapacksjob.js

```diff
@@ -501,8 +501,7 @@ DataPacksJob.prototype.formatResponse = async function(jobInfo, error) {
             response.message = jobInfo.message ?  jobInfo.message : jobInfo.hasError ? jobInfo.errors.join('\n').replace(/%vlocity_namespace%/g, this.vlocity.namespace) : `${jobInfo.jobAction} Successful`;
             response.status = jobInfo.hasError ? 'error' : 'success';
         } else if (jobInfo.jobAction === 'downloadPerformanceData') {
-            response.records = jobInfo.data;
-            response.message = jobInfo.errors.length != 0 ? jobInfo.errors.join('\n') : `Success - ${jobInfo.data.length} Records`;
+            response.records = [];
```

Comment:
Is this a correct change?

Reply:
Because it loads from the file anyway in the UI?

Reply:
Yes this is intentional. There were basically 2 reasons for "Refresh All" getting hanged up:
1. few undefined values is userIds while fetching UserDetails for performance
2. Passing large data to UI-->  it just hangs showing job complete message and control not coming to UI
Hopefully It should fix/improve the data loading